### ISSUE TITLE
Tracking QB changes

### ISSUE THREAD

**Issue by andr3w321** [BurntSushi/nflgame#148]
There are lots of QB injuries this week so I was interested in looking back at games where QB changes took place.  Anyone have a better way of doing it than this?  https://gist.github.com/andr3w321/99d64a99b303befca415  As you can see a lot of the players' teams is UNK.  You could probably combine with https://gist.github.com/ochawkeye/aa0aae0ec42c4ba6ca0e somehow to account for that.  The bye weeks are also unaccounted for.

**Comment by BurntSushi** [comment]
Yes, you will need to do it manually by looking at the team associated with the statistic recorded by a player in any given game. The API unfortunately cannot answer every question. :-)

To be clear: the player data contains *current* data. If you need historical data, you need to go and look at the actual play-by-play data.
